client/input.go

```diff
@@ -100,6 +95,19 @@ func GetConfirmation(prompt string, buf *bufio.Reader) (bool, error) {
 	}
 }
 
+// GetString simply returns the trimmed string output of a given reader.
+func GetString(prompt string, buf *bufio.Reader) (string, error) {
```

Comment:
This should probably be moved to a more generalized place? Or is this appropriate?

Reply:
I think this is a sufficiently general place given this file is primarily concerned with getting input from the user. Did you have a better place in mind?

Reply:
yeah you're right `client/input` is good 👍